core/src/main/java/org/eclipse/krazo/binding/convert/impl/BigDecimalConverter.java

```diff
@@ -46,7 +46,7 @@ public boolean supports(Class<BigDecimal> rawType) {
                             .orElse(null)
             );
 
-        } catch (ParseException e) {
+        } catch (ParseException | NumberFormatException e) {
```

Comment:
NumberFormat does not throw `NumberFormatException`, but the constructors of BigInteger and BigDecimal do. I didn't get the BigDecimalConverter to throw this, but BigIntegerConverter will, if you pass a decimal (like 12.3).

Reply:
Ah, good catch. Thanks a lot!

I wonder if we should improve `BigIntegerConverter` to handle the case of decimals better!? We could first create a `BigDecimal`, then round it to remove all fraction digits and then convert it to an `BitInteger`. If the user enters `12.9` and the value is bound to a `BigInteger`, binding a value of `13` (rounded via `RoundingMode.HALF_UP`) sounds reasonable to me. Thoughts?

Reply:
I support the idea of converting to BigInteger with loss of precision, but I'm not so sure about the rounding mode. This would mean that BigInteger behaves quiet different than Integer, which just drops the decimal, as long as it's in the precision:

```
jshell> (int) 2.5
$3 ==> 2

jshell> (int) 2.6
$4 ==> 2

jshell> (int) 2.9
$5 ==> 2

jshell> (int) 2.9999999999999999999999999
$6 ==> 3
```

Reply:
I would prefer using `RoundingMode.HALF_UP`. WDYT? 

Something like (pseudocode, no handling for empty optional yet):

```java
BigDecimal decimal = new BigDecimal( parseNumber(value, locale).toString() );
BigInteger result = new BigInteger( decimal.setScale(0, RoundingMode.HALF_UP).toString() );
```